.storybook/test-runner.ts

```diff
@@ -0,0 +1,29 @@
+import { injectAxe, checkA11y } from 'axe-playwright';
+import { Page } from 'playwright-core';
+import { parameters } from './preview';
+
+/*
+* See https://storybook.js.org/docs/react/writing-tests/test-runner#test-hook-api-experimental
```

Comment:
The indentation is a little off, I think we're missing a space before these asterisks.
I think the first first line should have two asterisks like `/**`

Reply:
If you type `/**` in vscode and hit enter it should automatically format stuff nicely for you, if you only type `/*` at least on my machine you don't get the formatting